warehouse/query-core/src/main/java/datawave/query/transformer/EdgeQueryTransformer.java

```diff
@@ -121,4 +121,5 @@ public EdgeBase transform(Entry<Key,Value> entry) {
         }
         return edge;
     }
+    
```

Comment:
??

Reply:
Removed -- it was leftover from moving the method into the AbstractQueryTransformer (and thus removed from child classes)